tests/images/customize.sh

```diff
@@ -58,6 +58,8 @@ opts_common=(
     --ignition-ca "${fixtures}/cert.pem"
     # Condition of @applied-live-ign@ and @applied-dest-ign@
     --network-keyfile "${fixtures}/installer-test.nmconnection"
+    # Condition of @applied-live-ign@ and @applied-dest-ign@
```

Comment:
This is not true currently, so you'll need to update the `live.ign` and `dest.ign` fixtures.

Reply:
After changing them to add the condition for generate NetworkManager keyfile I have try to run the tests with 
```bash
./tests/images.sh ~/Documents/cnv/sandbox/coreos-live/
```
And the output is the following, looks like the checksums does not match.
```bash
Boot media will automatically install to /dev/vda without confirmation.
1189: digest inplace-iso
222: sha256sum inplace-iso
222: awk '{print $1}'
189: '[' e41247f55606742bd0083c20ffc53bd92b8001d015165614f5858c37efb7b015 = fb49271eda68f2fa7ce2c4727411e3bfd30f099a93a3784d7469d4dedd63e54d ']'
1: rm -rf /tmp/tmp.sNbyWtpoKh
```

Do you know what's going on here ?

Reply:
I am going to proposa a PR a change some we can instruct it to generate the keyfile but without the UUID.